rkt/metadata_service.go

```diff
@@ -317,17 +317,18 @@ func handlePodAnnotations(w http.ResponseWriter, r *http.Request, pm *schema.Pod
 func handlePodAnnotation(w http.ResponseWriter, r *http.Request, pm *schema.PodManifest) {
 	defer r.Body.Close()
 
-	k, err := types.NewACName(mux.Vars(r)["name"])
+	n := mux.Vars(r)["name"]
+	k, err := types.NewACName(n)
```

Comment:
Good catch! seems we should have Annotations.Get() to take ACIdentifier instead of string.